src/Mod/Sketcher/Gui/TaskSketcherElements.cpp

```diff
@@ -57,6 +57,8 @@
 
 #include <Gui/Command.h>
 
+#include <QWidgetAction>

```

Comment:
Isn't this one precompiled?

Reply:
We had an issue with that for grid PR (command file). But it has been fixed.
But when I wrote that this morning in this PR it wasn't compiling when I put the include in the if.
I will try to change it again see if problem is gone.

Reply:
I just tested again, and if I put it in 
`#ifndef _PreComp_`
Then it doesn't compile.
It's strange because on the grid PR it was possible in the end after wmayer pushed a fix-commit. And this branch should be up to date so it should have this fix.

Reply:
in hte grid PR you are adding QWidgetAction to precompiled.h, in master it's not there yet AFAIK: https://github.com/FreeCAD/FreeCAD/pull/7754/files#diff-2eba49f3c71a2f076249b894e429c9907dbd7d6759749a3c259c59027db19b12

Reply:
Good catch. I forgot about that. Ok so after grid pr merge then we can move the header to the precomp if.